### ISSUE TITLE
Broken on SBCL-2.3.10

### ISSUE THREAD

**Issue by catap** [phoe/in-nomine#14]
```
; compiling file "/opt/local/var/macports/build/_Users_catap_src_macports-ports_lisp_cl-in-nomine/cl-in-nomine/work/build/source/cl-in-nomine/src/package.lisp" (written 15 NOV 2023 08:33:09 AM):

; file: /opt/local/var/macports/build/_Users_catap_src_macports-ports_lisp_cl-in-nomine/cl-in-nomine/work/build/source/cl-in-nomine/src/package.lisp
; in: UIOP/PACKAGE:DEFINE-PACKAGE #:IN-NOMINE
;     (UIOP/PACKAGE:DEFINE-PACKAGE #:IN-NOMINE (:USE #:CL #:ALEXANDRIA)
;                                  (:LOCAL-NICKNAMES (#:T #:TRIVIAL-ARGUMENTS))
;                                  (:EXPORT #:DEFINE-NAMESPACE #:CLEAR-NAMESPACE
;                                   #:NAMESPACE #:NAMESPACE-NAME
;                                   #:NAMESPACE-NAME-TYPE #:NAMESPACE-VALUE-TYPE
;                                   #:NAMESPACE-ACCESSOR #:NAMESPACE-CONDITION-NAME
;                                   #:NAMESPACE-TYPE-NAME
;                                   #:NAMESPACE-MAKUNBOUND-SYMBOL
;                                   #:NAMESPACE-BOUNDP-SYMBOL ...))
; 
; caught ERROR:
;   (during macroexpansion of (UIOP/PACKAGE:DEFINE-PACKAGE #:IN-NOMINE ...))
;   unrecognized define-package keyword :LOCAL-NICKNAMES

```

**Comment by phoe** [comment]
Upgrade UIOP to some version that includes https://gitlab.common-lisp.net/asdf/asdf/-/commit/85dedfebaf480a76325fa41e3bc50910b5c11f62

**Comment by phoe** [comment]
Fixed by [03524ba](https://github.com/phoe/in-nomine/commit/03524bab99309848d8a18de0673f055192058f0a)

**Comment by catap** [comment]
Confirmed, it works!
